src/coreclr/src/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.CoreCLR.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using Internal.Runtime.CompilerServices;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class MemoryMarshal
+    {
+        /// <summary>
+        /// Returns a reference to the 0th element of <paramref name="array"/>. If the array is empty, returns a reference to where the 0th element
+        /// would have been stored. Such a reference may be used for pinning but must never be dereferenced.
+        /// </summary>
+        /// <exception cref="NullReferenceException">If <paramref name="array"/> is <see langword="null"/>.</exception>
```

Comment:
doc nit:
```suggestion
        /// <exception cref="NullReferenceException"><paramref name="array"/> is <see langword="null"/>.</exception>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions#how-to
> Write each condition as if the word "if" precedes it.